src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,89 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Entry<K, V>[] table;
```

Comment:
The field 'table' should be initialized using the diamond operator to ensure type safety. For example: 'new Entry<K, V>[INITIAL_CAPACITY];'. However, due to type erasure, you will need to cast it, like this: '(Entry<K, V>[]) new Entry<?, ?>[INITIAL_CAPACITY];'.